iipsrv/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3
+
+RUN apk update
+RUN apk upgrade
+RUN apk add --no-cache \
+  autoconf \
+  automake \
+  build-base \
+  jpeg-dev \
+  libtool \
+  m4 \
+  make \
+  pkgconfig \
+  tiff-dev
+
+RUN wget https://github.com/ruven/iipsrv/archive/refs/tags/iipsrv-1.1.zip
+RUN unzip iipsrv-1.1.zip
```

Comment:
just a note that IIP 1.1 was released in 2019, and there have been quite a few improvements since then. Generally it's OK to run off the master branch, but you could also run off a specific commit hash to make sure you're not automatically updating the code.